crates/ruff_linter/src/rules/flake8_implicit_str_concat/snapshots/ruff_linter__rules__flake8_implicit_str_concat__tests__ISC003_ISC.py.snap

```diff
@@ -1,34 +1,67 @@
 ---
 source: crates/ruff_linter/src/rules/flake8_implicit_str_concat/mod.rs
 ---
-ISC.py:9:3: ISC003 Explicitly concatenated string should be implicitly concatenated
+ISC.py:9:3: ISC003 [*] Explicitly concatenated string should be implicitly concatenated
    |
  8 |   _ = (
  9 | /   "abc" +
 10 | |   "def"
    | |_______^ ISC003
 11 |   )
    |
+   = help: Remove redundant '+' operator to implicitly concatenate
 
-ISC.py:14:3: ISC003 Explicitly concatenated string should be implicitly concatenated
+ℹ Safe fix
+6  6  |     "def"
+7  7  | 
+8  8  | _ = (
+9     |-  "abc" +
+   9  |+  "abc" 
+10 10 |   "def"
+11 11 | )
+12 12 | 
+
+ISC.py:14:3: ISC003 [*] Explicitly concatenated string should be implicitly concatenated
    |
 13 |   _ = (
 14 | /   f"abc" +
 15 | |   "def"
    | |_______^ ISC003
 16 |   )
    |
+   = help: Remove redundant '+' operator to implicitly concatenate
+
+ℹ Safe fix
+11 11 | )
+12 12 | 
+13 13 | _ = (
+14    |-  f"abc" +
+   14 |+  f"abc" 
```

Comment:
It seems that the fix failes to remove the trailing whitespace. Can you try to remove it too?